File: NodeJS/selectors/select_by_css.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  const element = await page.$('h1');
  console.log('CSS Selector:', await element.textContent());
  await browser.close();
})();

File: Python/selectors/select_by_css.py
from playwright.sync_api import sync_playwright

def run():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page()
        page.goto("https://example.com")
        element = page.query_selector("h1")
        print("CSS Selector:", element.inner_text())
        browser.close()

run()